### DISCUSSION TITLE
Adding Surfaces to Optics

### DISCUSSION THREAD

**Discussion by scriswell** [RoleModel/optics#180] [Ideas]
https://codepen.io/Jeremy-Walton/pen/xxQENzo

**Comment by scriswell** [comment]
We need to investigate surface patterns to further unlock design edge cases and simplify theming within Optics. The attached code pen demonstrates some early explorations.

**Comment by Jeremy-Walton** [comment]
Another component idea is a title pair (or some other name). If implemented, it would definitely need the Surfaces idea to be effective. https://codepen.io/Jeremy-Walton/pen/yLQbPWq?editors=1100
